src/Compilers/CSharp/Test/Symbol/Compilation/CompilationAPITests.cs

```diff
@@ -2053,6 +2053,30 @@ public void PreviousSubmissionWithError()
             Assert.Throws<InvalidOperationException>(() => CreateSubmission("a + 1", previous: s0));
         }
 
+        [Fact]
+        public void CreateTupleTypeSymbol()
```

Comment:
The case where `Compilation` doesn't contain `ValueTuple` is what is tested here. Your question makes me think that the tuple should be error type if the underlying type is error. I will change and test that. 
I think this also goes in the direction of unifying tuple and underlying type.